src/generated/resources/assets/railways/lang/zh_cn.json

```
@@ -0,0 +1,91 @@
+	"item.railways.green_incomplete_conductor_cap": "绿色列车长帽（半成品）",
+	"item.railways.jukeboxcart": "唱片机矿车",
+	"item.railways.light_blue_conductor_cap": "淡蓝色列车长帽",
+	"item.railways.light_blue_incomplete_conductor_cap": "淡蓝色列车长帽（半成品）",
+	"item.railways.light_gray_conductor_cap": "淡灰色列车长帽",
+	"item.railways.light_gray_incomplete_conductor_cap": "淡灰色列车长帽（半成品）",
+	"item.railways.lime_conductor_cap": "黄绿色列车长帽",
+	"item.railways.lime_incomplete_conductor_cap": "黄绿色列车长帽（半成品）",
+	"item.railways.magenta_conductor_cap": "品红色列车长帽",
+	"item.railways.magenta_incomplete_conductor_cap": "品红色列车长帽（半成品）",
+	"item.railways.orange_conductor_cap": "橙色列车长帽",
+	"item.railways.orange_incomplete_conductor_cap": "橙色列车长帽（半成品）",
+	"item.railways.pink_conductor_cap": "粉红色列车长帽",
+	"item.railways.pink_incomplete_conductor_cap": "粉红色列车长帽（半成品）",
+	"item.railways.purple_conductor_cap": "紫色列车长帽",
+	"item.railways.purple_incomplete_conductor_cap": "紫色列车长帽（半成品）",
+	"item.railways.red_conductor_cap": "红色列车长帽",
+	"item.railways.red_incomplete_conductor_cap": "红色列车长帽（半成品）",
+	"item.railways.track_incomplete_acacia": "金合欢木列车轨道（半成品）",
+	"item.railways.track_incomplete_birch": "白桦木列车轨道（半成品）",
+	"item.railways.track_incomplete_crimson": "绯红木列车轨道（半成品）",
+	"item.railways.track_incomplete_dark_oak": "深色橡木列车轨道（半成品）",
+	"item.railways.track_incomplete_jungle": "丛林木列车轨道（半成品）",
+	"item.railways.track_incomplete_oak": "橡木列车轨道（半成品）",
+	"item.railways.track_incomplete_spruce": "云杉木列车轨道（半成品）",
+	"item.railways.track_incomplete_warped": "诡异木列车轨道（半成品）",
+	"item.railways.white_conductor_cap": "白色列车长帽",
+	"item.railways.white_incomplete_conductor_cap": "白色列车长帽（半成品）",
+	"item.railways.yellow_conductor_cap": "黄色列车长帽",
+	"item.railways.yellow_incomplete_conductor_cap": "黄色列车长帽（半成品）",
+
+	"itemGroup.railways": "机械动力：汽鸣铁道",
+
+	"railways.ponder.conductor_constructing.header": "创建一个指挥机器人",
+	"railways.ponder.conductor_constructing.text_1": "指挥机器人由安山机壳组成。",
+	"railways.ponder.conductor_redstoning.header": "借助指挥机器人远程操控红石元件",
+	"railways.ponder.conductor_redstoning.text_1": "指挥机器人可以在玩家注视它时按下按钮。",
+	"railways.ponder.conductor_redstoning.text_2": "它同样可以在玩家注视它时拉动拉杆。",
+	"railways.ponder.conductor_redstoning.text_3": "当玩家戴着与其颜色不一致的列车长帽时，指挥机器人不会激活红石元件。",
+	"railways.ponder.conductor_toolboxing.header": "使用指挥机器人运输工具箱",
+	"railways.ponder.conductor_toolboxing.text_1": "工具箱装备在指挥机器人身上时，其功能正常",
+	"railways.ponder.train_semaphore.header": "使用壁板信号机可视化信号状态",
+	"railways.ponder.train_semaphore.text_1": "将壁板信号机放置在列车信号机上方的杆上",
+	"railways.ponder.train_semaphore.text_2": "壁板信号机下方的支柱可由不同材料组成",
+	"railways.ponder.train_semaphore.text_3": "壁板信号机会实时显示支柱下信号机的信号状态",
+	"railways.ponder.train_semaphore.text_4": "如果黄铜信号区间有多个有效的出口路径...",
+	"railways.ponder.train_semaphore.text_5": "...可以将第二个壁板信号机放置在第一个的上方",
+	"railways.ponder.train_semaphore.text_6": "如果有任一可用信号区间被阻塞，底部的信号机将会关闭",
+	"railways.ponder.train_semaphore.text_7": "如果所有信号区间都被阻塞，顶部的信号机将会关闭",
+	"railways.ponder.train_semaphore.text_8": "当两个壁式信号机都放置在非黄铜信号机上时，它们会同时关闭",
```

```suggestion
	"railways.ponder.train_semaphore.header": "使用臂板信号机可视化信号状态",
	"railways.ponder.train_semaphore.text_1": "将臂板信号机放置在列车信号机上方的杆上",
	"railways.ponder.train_semaphore.text_2": "臂板信号机下方的支柱可由不同材料组成",
	"railways.ponder.train_semaphore.text_3": "臂板信号机会实时显示支柱下信号机的信号状态",
	"railways.ponder.train_semaphore.text_4": "如果黄铜模式信号区间有多个有效的出口路径……",
	"railways.ponder.train_semaphore.text_5": "……可以将第二个臂板信号机放置在第一个的上方",
	"railways.ponder.train_semaphore.text_6": "如果有任一可用区间被阻塞，底部的臂板信号机将会关闭",
	"railways.ponder.train_semaphore.text_7": "如果所有区间都被阻塞，顶部的臂板信号机将会关闭",
	"railways.ponder.train_semaphore.text_8": "当两个臂板信号机都放置在非黄铜信号机上时，它们会同时关闭",
```
wrong character & proper ellipsis